integration/kafka/src/main/java/alluxio/kafka/connect/AlluxioTopicPartitionWriter.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.FileDoesNotExistException;
+import alluxio.kafka.connect.format.AlluxioFormat;
+
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.sink.SinkTaskContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Queue;
+import java.util.UUID;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * This class writes kafka's specific topic partition data to alluxio filesystem.
+ */
+public class AlluxioTopicPartitionWriter {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AlluxioSinkTask.class);
+
```

Comment:
also for those member variables, if they are only assigned in constructor and never changed, we typically mark them `final`; otherwise if they are possibly changed in other methods, leave them as is.